packages/styled-ppx/styled-ppx.0.57.0/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+synopsis: "Type-safe styled components for ReScript and Melange"
+description:
+  "styled-ppx is the ppx that brings styled components to ReScript and Melange, allowing you to create React Components with type-safe style definitions using CSS."
+maintainer: ["David Sancho <dsnxmoreno@gmail.com>"]
+authors: ["David Sancho <dsnxmoreno@gmail.com>"]
+license: "MIT"
+homepage: "https://styled-ppx.vercel.app"
+bug-reports: "https://github.com/davesnx/styled-ppx/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml" {>= "5.1.0"}
+  "reason" {>= "3.11.0"}
+  "menhir" {>= "20220210"}
+  "ppx_deriving" {>= "5.0"}
+  "ppx_deriving_yojson" {>= "3.7.0"}
+  "ppxlib" {>= "0.27.0"}
+  "sedlex" {>= "3.2"}
+  "melange" {>= "3.0.0"}
+  "server-reason-react"
+  "reason-react" {>= "0.14.0"}
+  "alcotest" {with-test}
```

Comment:
```suggestion
  "alcotest"
```

this is currently not just a test dependency, is it expected?
```
#=== ERROR while compiling styled-ppx.0.57.0 ==================================#
# context              2.2.0~rc1 | linux/x86_64 | ocaml-base-compiler.5.2.0 | pinned(https://github.com/davesnx/styled-ppx/releases/download/0.57.0/styled-ppx-0.57.0.tbz)
# path                 ~/.opam/5.2/.opam-switch/build/styled-ppx.0.57.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p styled-ppx -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/styled-ppx-7-bd5418.env
# output-file          ~/.opam/log/styled-ppx-7-bd5418.out
### output ###
# File "packages/alcotest-extra/dune", line 4, characters 12-20:
# 4 |  (libraries alcotest))
#                 ^^^^^^^^
# Error: Library "alcotest" not found.
# -> required by library "styled-ppx.alcotest-extra" in
#    _build/default/packages/alcotest-extra
# -> required by _build/default/META.styled-ppx
# -> required by _build/install/default/lib/styled-ppx/META
# -> required by _build/default/styled-ppx.install
# -> required by alias install
```

Reply:
oh shit, I made a library depending on alcotest but didn't plan to make it "public". Let me push a new version with a fix for this.